hack/generator/pkg/codegen/pipeline/testdata/TestInjectPropertyAssignmentFunctions-v20200101.golden

```diff
@@ -3,159 +3,36 @@
 // Licensed under the MIT license.
 package v20200101
 
-import (
-	"github.com/Azure/azure-service-operator/testing/microsoft.person/v20200101storage"
-	"github.com/pkg/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+//Storage version of v20200101.Person
 type Person struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	Spec              Person_Spec   `json:"spec,omitempty"`
 	Status            Person_Status `json:"status,omitempty"`
 }
 
-// AssignPropertiesFromPerson populates our Person from the provided source Person
```

Comment:
Why did these go away? Aren't they still needed for conversion to storage versions?

Reply:
These have no longer changed either.